modules/carto/src/sources/boundary-query-source.ts

```diff
@@ -3,26 +3,26 @@ import {baseSource} from './base-source';
 import type {SourceOptions, TilejsonResult} from './types';
 
 export type BoundaryQuerySourceOptions = SourceOptions & {
-  boundaryId: string;
+  tilesetTableName: string;
   matchingColumn?: string;
-  sqlQuery: string;
+  propertiesSqlQuery: string;
   queryParameters?: QueryParameters;
 };
 type UrlParameters = {
-  boundaryId: string;
+  tilesetTableName: string;
   matchingColumn: string;
-  sqlQuery: string;
+  sqlQuery: string; // TODO rename to propertiesSqlQuery
```

Comment:
Why is this TODO? You've changed the`boundaryId` to `tilesetTableName` so API changes are required anyway